tests/lang_defs/distinct/tdistinct.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  targets: "c js"
+  targets: "c js vm"
```

Comment:
Note to self: have a look at the `lang_defs` category and see if it makes sense to just default the whole thing to all targets.